templates/search/blocks/search_results__selected-filters.html

```diff
@@ -7,7 +7,7 @@ <h2 class="sr-only">Selected filters</h2>
         {% for field_name, selected_options in selected_filters.items %}
```

Comment:
Can `selected_filters.items` not give us values that are fit for direct display? The template shouldn't need to have all of these special cases for individual fields.

Reply:
Nope, both servers its own purpose.